torch/testing/_internal/common_methods_invocations.py

```diff
@@ -524,10 +524,13 @@ def __init__(self,
                  error_inputs_func=None,  # function to generate inputs that will throw errors
 
                  # the following metadata relates to dtype support and is tested for correctness in test_ops.py
-                 dtypes=floating_types(),  # dtypes this function is expected to work with
+                 dtypes,  # dtypes this function works with on the CPU,
```

Comment:
Thanks for @pmeier for the suggestion of making it without default. We can now get away with `assert dtypes is not None`